metacatalog/ext/standards_export/schemas/datacite/datacite.j2

```diff
@@ -0,0 +1,83 @@
+<resource xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="http://datacite.org/schema/kernel-4"
+    xsi:schemaLocation="http://datacite.org/schema/kernel-4 http://schema.datacite.org/meta/kernel-4.3/metadata.xsd">
+    <identifier identifierType="UUID">{{uuid}}</identifier>
+    {% for author in authors %}
+    <creators>
+        <creator>
+            {% if author.last_name %}
+            <creatorName nameType="Personal">{{author.last_name}}, {{author.first_name}}</creatorName>
+            <givenName>{{author.first_name}}</givenName>
+            <familyName>{{author.last_name}}</familyName>            
+            {% else %}
+            <creatorName nameType="Organizational">{{author.organisation_name}}</creatorName>
+            {% endif %}
+            {% if author.affiliation %}
+            <affiliation>{{author.affiliation}}</affiliation>
+            {% endif %}
+        </creator>
+    </creators>
+    {% endfor %}
+    <titles>
+        <title>{{title}}</title>
+    </titles>
+    <publisher>{{contact.organisationName}}</publisher>
+    <publicationYear>{{publication[0:4]}}</publicationYear>
+    {% if keywords or details%}
+    <subjects>
+        {% for detail in details %}
+        <subject>{{detail}}</subject>
+        {% endfor %}
+        {% for keyword in keywords %}
+        <subject subjectScheme="{{keyword.thesaurusName}}">{{keyword.full_path}}</subject>
+        {% endfor %}
+    </subjects>
+    {% endif %}
+    <contributors>
+        <contributor contributorType="ContactPerson">
+            <contributorName nameType="Organizational">{{contact.organisationName}}</contributorName>
+        </contributor>
+    </contributors>
```

Comment:
add the first author as another contributor in the role of a contact person

Reply:
I removed us (V-For-WaTer) from `<contributors` and only added the first author as `ContactPerson`.

V-For-WaTer would fit as `contributorType="HostingInstitution"`, but V-For-WaTer is already the `<publisher>` of the resource. According to the [DataCite documentation](https://support.datacite.org/docs/datacite-metadata-schema-v44-recommended-and-optional-properties#:~:text=to%20the%20name.-,HostingInstitution,-Typically%2C%20the%20organisation), we do not put the publisher into contributors again.